llvm/test/CodeGen/NVPTX/div.ll

```diff
@@ -0,0 +1,16 @@
+; RUN: llc < %s -march=nvptx64 | FileCheck %s
```

Comment:
Generating checks with utils/update_llc_test_checks.py would be a good idea.

Reply:
Thanks @Artem-B for the suggestion! I have updated the test by running `utils/update_llc_test_checks.py`. Can I go ahead with the merge?

Reply:
Go ahead.